build_related_scripts/get_modified_packs.py

```diff
@@ -41,13 +45,17 @@ def get_changed_files(repo_path: Path, prev_ver: str) -> List[str]:
     Returns:
         List[str]. All the files that have changed.
     """
-    repo = Repo(repo_path, search_parent_directories=True)
     git_util = GitUtil(repo_path)
+    repo = git_util.repo
+
+    try:
+        active_branch = repo.active_branch
```

Comment:
maybe it will be easier to use commit instead of branch?
The commit is availible also in detached head state.